### PR TITLE
build: AUR automated release

### PR DESCRIPTION
**Pull request by morrieinmaas**
Github action/pipeline to automate releasing new versions to AUR

### CODE REVIEW

File: .github/workflows/aur-release.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        run: |
+          # get the version
+          VERSION=$(curl --silent https://api.github.com/repos/animo/aries-cli/releases | jq '.[-1]' |  jq -r .tag_name)
+          # get the aur repo
+          git clone ssh://aur@aur.archlinux.org/aries-cli.git
+          # overwrite with the new files
+          cp PKGBUILD aries-cli/PKGBUILD
+          cp .SRCINFO aries-cli/.SRCINFO
+          cd aries-cli
+          # use the rsa key for git
```
Comment (blu3beri) [inline .github/workflows/aur-release.yml:40]: We should use ed25519 ;)

**Review by blu3beri — APPROVED** [review]

**Comment by blu3beri** [comment]
Looks great! Just fix the commits, remove unrelated ones, and we can merge.

File: .github/workflows/aur-release.yml (hunk 1 above)
Comment (morrieinmaas) [reply .github/workflows/aur-release.yml:40]: alright i'll change that

File: .github/workflows/aur-release.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+jobs:
+  release-brew:
+    name: release brew
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: Get PKGBUILD
+        id: get_pkgbuild
+        run: curl -o PKGBUILD https://raw.githubusercontent.com/animo/aries-cli/main/pkg/PKGBUILD
+      - name: Gather data and rewrite ruby file
```
Comment (blu3beri) [inline .github/workflows/aur-release.yml:21]: Ruby?

**Comment by morrieinmaas** [comment]
thanks. well it's branched off the other one - thought that'd be merged already. I'll get rid of the junk ;)

File: .github/workflows/aur-release.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    tags:
+      - "v*.*.*"
+  workflow_run:
+    workflows: ["build-and-release"]
+    types: 
+      - completed
+
+jobs:
+  release-brew:
+    name: release brew
```
Comment (blu3beri) [inline .github/workflows/aur-release.yml:14]: Brew

File: .github/workflows/aur-release.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  push:
+    tags:
+      - "v*.*.*"
+  workflow_run:
+    workflows: ["build-and-release"]
+    types: 
+      - completed
+
+jobs:
+  release-brew:
```
Comment (blu3beri) [inline .github/workflows/aur-release.yml:13]: Brew

**Review by blu3beri — CHANGES_REQUESTED** [review]

File: .github/workflows/aur-release.yml (hunk 2 above)
Comment (morrieinmaas) [reply .github/workflows/aur-release.yml:21]: lol - sorry about that should really call it a day now 😅

**Comment by morrieinmaas** [comment]
The other PR is good to go. Wait with this please, I wanna test it a bit more after some sleep

File: .github/workflows/aur-release.yml (hunk 1 above)
Comment (blu3beri) [reply .github/workflows/aur-release.yml:40]: Haha its okay. Was more a joke because we use that for signatures in AFJ :).

**Comment by blu3beri** [comment]
You can mention me when you are done. If everything works it looks good to me :).

**Comment by morrieinmaas** [comment]
tested all the steps from local and it that works. However there is always a chance if container of gh action behaving slightly different. let me trigger this ofr the current build and see if it works

**Comment by morrieinmaas** [comment]
Just FYI @blu3beri (as you probably have noticed already) it's not yet working using the GH actions runner. I'm working on it. Just trying to debug this - felling like a monkey with a drum right now

**Comment by morrieinmaas** [comment]
@blu3beri I'm stuck atm. The cert permissions on the runner give me errors although I change them. Not quite sure how to solve this. Will have another look.

**Comment by morrieinmaas** [comment]
alrighty @blu3beri testest in [fork](https://github.com/morrieinmaas/agent-cli/actions/runs/2245609058) and works finally. thx for your help

**Review by blu3beri — APPROVED** [review]

**Pull request merged**